src/App.tsx

```diff
@@ -78,23 +100,36 @@ const App = () => {
     <>
       <Routes>
         <Route path="/" element={<Home />} />
-        <Route path="/tasks" element={<ActiveTasks />} />
+        <Route
+          path="/tasks"
+          element={<ActiveTasks activeTasks={activeTasksList} />}
+        />
         <Route
           path="/calendar"
-          element={
-            <Calendar
-              completedTasks={completedTaskArray}
-              changeDate={changeDate}
-              date={date}
-            />
-          }
+          element={<Calendar completedTasks={completedTasks} date={date} />}
         />
+        <Route
+          path="/leaderboard"
+          element={<Leaderboard users={fetchedTribe} />}
+        />
+        <Route path="/profile" element={<Profile user={tribeUsers[0]} />} />
+        path="/calendar" element=
```

Comment:
You should be able to delete this code between the { } I think this is from the merge conflict.